api/core/model_runtime/model_providers/bedrock/llm/llm.py

```diff
@@ -90,7 +90,7 @@ def _code_block_mode_wrapper(
         stop: Optional[list[str]] = None,
         stream: bool = True,
         user: Optional[str] = None,
-        callbacks: list[Callback] = None,
+        callbacks: Optional[list[Callback]] = None,
```

Comment:
This parameter is unused, you can remove it.

Reply:
(ditto) Will remove the unused parameter in the follow-up PR, covering the method's signature and usages.